Ready for year 2038

Alkarex opened the pull request:
Fix https://github.com/FreshRSS/FreshRSS/discussions/5569
Requires PHP on a 64-bit platform to take advantage of it.
https://en.wikipedia.org/wiki/Year_2038_problem
Note: at the first refresh for a user, it will take some time to alter the column types, but meanwhile, the other users are not blocked.

Alkarex committed:
Ready for year 2038
Fix https://github.com/FreshRSS/FreshRSS/discussions/5569
Requires PHP on a 64-bit platform to take advantage of it.
https://en.wikipedia.org/wiki/Year_2038_problem

Frenzie reviewed (approved):
I'm somewhat surprised PostgreSQL was different from the others here.

Alkarex committed:
Allows dates past 2038
Rework of https://github.com/FreshRSS/FreshRSS/pull/3259
https://github.com/FreshRSS/FreshRSS/issues/3258

Alkarex committed:
Merge branch 'edge' into date-2038

Alkarex committed:
Auto alter columns

Alkarex committed:
Merge branch 'edge' into date-2038

Alkarex committed:
Changelog
